pkg/terminal/command.go

```diff
@@ -1032,13 +1048,27 @@ func (c *Commands) next(t *Term, ctx callContext, args string) error {
 	if c.frame != 0 {
 		return notOnFrameZeroErr
 	}
-	state, err := exitedToError(t.client.Next())
-	if err != nil {
-		printcontextNoState(t)
+	var count int64
+	var err error
+	if count, err = parseOptionalCount(args, 1); err != nil {
 		return err
 	}
-	printcontext(t, state)
-	return continueUntilCompleteNext(t, state, "next")
+	if count != 0 {
+		var state *api.DebuggerState
+		for ; count != 0; count-- {
+			state, err = exitedToError(t.client.Next())
+			if err != nil {
+				break
```

Comment:
just call `printcontextNoState` and return here instead of breaking.

Reply:
ok!